docs/source/install/deb.rst

```diff
@@ -35,7 +45,6 @@ Install StackStorm components
 
   .. code-block:: bash
 
-      sudo apt-get update
```

Comment:
Removed it, because `apt-get update` was done just in previous step, after adding Bintray repo.
Considering it's a fresh install, 2 update commands in a row are excessive.